src/dms/tests/EdFi.DataManagementService.Tests.E2E/Management/OpenSearchContainerSetup.cs

```diff
@@ -35,9 +35,17 @@ private static async Task ResetOpenSearch()
         OpenSearchClient openSearchClient = new();
         var indices = await openSearchClient.Cat.IndicesAsync();
 
-        foreach (var index in indices.Records.Where(x => x.Index.Contains("ed-fi")))
+        foreach (var index in indices.Records.Where(x => x.Index.Contains("ed-fi$")))
         {
             await openSearchClient.Indices.DeleteAsync(index.Index);
+
+            // Recreate the index with default or custom settings if needed
+            await openSearchClient.Indices.CreateAsync(index.Index, c => c
+                .Settings(s => s
+                    .NumberOfShards(1)
+                    .NumberOfReplicas(1)
+                )
+            );
```

Comment:
After a lot of testing I identified that ElasticSearch does not like indexes to be deleted, so I tried re-creating them but empty.